data/json/itemgroups/guns.json

```diff
@@ -159,6 +159,7 @@
       [ "fn_fal", 40],
       [ "hk_g3", 40 ],
       [ "hk_g36", 30 ],
+      [ "henry_big_boy", 30],
```

Comment:
In that grouping, yes. I don't want to rebalance all of guns for the sake of this PR